letsencrypt/cli.py

```diff
@@ -657,6 +730,10 @@ def create_parser(plugins, args):
         None, "-t", "--text", dest="text_mode", action="store_true",
         help="Use the text output instead of the curses UI.")
     helpful.add(None, "-m", "--email", help=config_help("email"))
+    helpful.add(None, "--apache", action="store_true",
+                help="Obtain and install certs using Apache")
+    helpful.add(None, "--nginx", action="store_true",
```

Comment:
why even bother if we decided to drop nginx plugin from `letsencrypt-auto` installation?